packages/aila/src/core/chat/AilaChat.ts

```diff
@@ -233,13 +233,11 @@ export class AilaChat implements AilaChatService {
 
   private accumulatedText() {
     const accumulated = this._chunks?.join("");
-    return accumulated;
+    return accumulated ?? "";
```

Comment:
Does this have any unexpected effects? I could imagine an ||= somewhere!

Reply:
No effects, everywhere just checks is result is truthy.